notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -580,7 +503,7 @@
         "id": "k_QorXXztzPH"
       },
       "source": [
-        "Start a new experiment run to track training parameters and start the training job. Note that this operation will take around 10 mins."
+        "Start a new experiment run to track training parameters and start the training job. Note that this operation  take around 10 mins."
```

Comment:
remove extra space and change "take" to "takes" (to agree with the singular subject):

- From: Note that this operation  take around 10 mins.
- To: Note that this operation takes around 10 mins.